libbpf-rs/src/map.rs

```diff
@@ -211,6 +214,56 @@ impl Map {
         })
     }
 
+    /// Open a previously pinned map from it's path.
+    ///
+    /// # Panics
+    /// If the path contains null bytes.
+    pub fn from_pinned_path<P: AsRef<Path>>(path: P) -> Result<Self> {
```

Comment:
Would it be possible to add a test for this constructor so that we can see its usage?